.github/workflows/ci.yaml

```
@@ -0,0 +1,37 @@
+    - name: Checkout Actions Repository
+      uses: actions/checkout@v4
+
+    - uses: crate-ci/typos@master
+      with:
+        files: '*.typ'
+        config: ./typos.toml
+
+  secretlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

yamllint

[warning] wrong indentation: expected 4 but found 6

(indentation)